COVID19/src/screens/TestType.js

```diff
@@ -12,14 +12,14 @@ const TestType = ({navigation}) => {
       <Text style={styles.txtTitle}> {constans.testType} </Text>
       <ButtonMain
         captionButton={constans.viralTest}
-        img={require('../utils/images/covidsitos.png')}
+        // img={require('../utils/images/covidsitos.png')}
```

Comment:
remove this comment if is not neccessary

Reply:
fixed